### PR TITLE
docs: Add 2023-03-09 meeting notes

### LINKED ISSUE #414: TSC meeting 09-March-2023
# Time

UTC Thu 09-Mar-2023 21:00:
- Los Angeles: Thu 09-Mar-2023 13:00
- Chicago: Thu 09-Mar-2023 15:00
- New York: Thu 09-Mar-2023 16:00
- Madrid: Thu 09-Mar-2023 22:00
- Moscow: Fri 10-Mar-2023 00:00
- Tokyo: Fri 10-Mar-2023 06:00
- Sydney: Fri 10-Mar-2023 08:00

# Location

https://eslint.org/chat/tsc-meetings

# Agenda

Extracted from:

* Issues and pull requests from the ESLint organization with the ["tsc agenda" label](https://github.com/issues?utf8=%E2%9C%93&q=org%3Aeslint+label%3A%22tsc+agenda%22)
* Comments on this issue

# Invited

- Nicholas C. Zakas (@nzakas) - TSC
- Milos Djermanovic (@mdjermanovic) - TSC

# Public participation

Anyone is welcome to attend the meeting as observers. We ask that you refrain from interrupting the meeting once it begins and only participate if invited to do so.

### PR DESCRIPTION
**Pull request by sam3k**
Closes #414

### CODE REVIEW

File: notes/2023/2023-03-09.md
Comment (nzakas) [inline notes/2023/2023-03-09.md:5]: The transcript format is always the same, so you can put it in and trust that it will be uploaded eventually. :)

```suggestion
[`2023-03-09-transcript.md`](2023-03-09-transcript.md)
```

File: notes/2023/2023-03-09.md
Comment (nzakas) [inline notes/2023/2023-03-09.md:36]:
```suggestion
> @nzakas:
```

File: notes/2023/2023-03-09.md
Comment (nzakas) [inline notes/2023/2023-03-09.md:41]: Let's call out when there's an action item and who is responsible for it.

```suggestion
**Resolution:** Okay, we've agreed to license the docs under Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. We will place a notice both in the docs README and in the footer of the docs site.

**Action Item:** @Sam3k to create an issue for this.
```

**Review by nzakas — DISMISSED** [review]
This looks good! Just left a couple of suggestions.

Please be sure to follow up on creating the issue we mentioned, and on updating the RFC PR to remove the "tsc agenda" label and add the summary we discussed over Discord.

Thanks!

**Comment by sam3k** [comment]
@nzakas:

- Accepted all your suggestions. Thank you!
- Just removed the `tsc-agenda` from [RFC](https://github.com/eslint/rfcs/pull/99#issuecomment-1465354131)
- Left a TSC Summary comment in [RFC](https://github.com/eslint/rfcs/pull/99#issuecomment-1465354131)
- Added `Final Commenting` label to [RFC](https://github.com/eslint/rfcs/pull/99#issuecomment-1465354131)
- The issue has been [created](https://github.com/eslint/eslint/issues/16980) as well

Let me know if I've missed anything.

**Review by nzakas — APPROVED** [review]
LGTM. Just waiting for @mdjermanovic to approve before merging.

**Review by mdjermanovic — APPROVED** [review]
Thanks!

**Pull request merged**
